modules/eido/validate/main.nf

```diff
@@ -0,0 +1,37 @@
+process EIDO_VALIDATE {
+    tag '$samplesheet'
+    label 'process_low'
+
+    conda (params.enable_conda ? "conda-forge::eido=0.1.8" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://containers.biocontainers.pro/s3/SingImgsRepo/eido/0.1.9_cv1/eido_0.1.9_cv1.sif' :
+        'biocontainers/eido:0.1.9_cv1' }"
+
+    input:
+    path samplesheet
+    path schema
+
+    output:
+    path "versions.yml"           , emit: versions
+    path "conversion.out"         , emit: conversion
```

Comment:
```suggestion
    path "*.log"         , emit: log
```

`validate` doesn't actually convert right? Or did you mean 'for' conversion (I found this a bit confusing if so)